core/src/main/scala/flawless/syntax/syntax.scala

```diff
@@ -43,8 +58,10 @@ object dsl {
         .map(_.toList.toNel.getOrElse(NonEmptyList.one(Assertion.Successful)))
     }
 
-  def pureTest(name: String)(assertions: NonEmptyList[Assertion]): NonEmptyList[Test[Nothing]] =
+  def pureTest(name: String)(assertions: NonEmptyList[Assertion])(implicit deps: CallsiteDeps): NonEmptyList[Test[Nothing]] = {
+    println(deps)
```

Comment:
This is just to prove the implicit is instantiated at the correct location (which is, at the call site of the `pureTest` function and not in line 61 of syntax.scala)